sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformation.scala

```diff
@@ -200,33 +286,43 @@ private[hive]
 case class HiveScriptIOSchema (
```

Comment:
The changes in this class are purely code-cleanup, mostly related to making more variables private and performing some cleanup related to nullability.